config/locales/modules/bravo.yml

```diff
@@ -16,7 +16,7 @@ en:
         heading: text_page
         body: page 1 of sub 1 (topic 1)
       1-1-2:
-        heading: youtube_page
+        heading: video_page
         body: page 2 of sub 1 (topic 2)
```

Comment:
What happens if a member of the content team omits the video attributes as is done here? Does it fail gracefully and can we improve it with any feedback?

Reply:
Maybe an `if/else` in the template checking for `:video`. If it is missing the bulk of the page becomes a big placeholder?